tasks/src/main/java/ru/sberbank/school/task02/calculator/CompareQuotesBeneficiary.java

```diff
@@ -0,0 +1,32 @@
+package ru.sberbank.school.task02.calculator;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+
+import java.util.Comparator;
+
+public class CompareQuotesBeneficiary implements Comparator<QuotePrice> {
+    private Beneficiary beneficiary;
+    private ClientOperation operation;
+
+    public CompareQuotesBeneficiary(ClientOperation operation, Beneficiary beneficiary) {
+        this.beneficiary = beneficiary;
+        this.operation = operation;
+    }
+
+    @Override
+    public int compare(QuotePrice quote1, QuotePrice quote2) {
+        //Ищем большую цену продажи банком//Offer
+        if ((beneficiary == Beneficiary.BANK && operation == ClientOperation.BUY)
+        || (beneficiary == Beneficiary.CLIENT && operation == ClientOperation.SELL)){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2036745562>*